actors/power/src/lib.rs

```diff
@@ -364,6 +367,32 @@ impl Actor {
         })
     }
 
+    /// Returns the total raw power of the network.
+    fn network_raw_power(rt: &mut impl Runtime) -> Result<NetworkRawPowerReturn, ActorError> {
+        rt.validate_immediate_caller_accept_any()?;
+        let st: State = rt.state()?;
+
+        Ok(NetworkRawPowerReturn { raw_byte_power: st.total_raw_byte_power })
+    }
+
+    /// Returns the raw power of the specified miner.
```

Comment:
Perhaps use the term "claimed" by the miner?

Note in docs that this does not include calculation of whether the miner meeds the minimum commitment to have any consensus power. The sum of miner_raw_power for all miners does not equal network_raw_power for this reason.

I think we should probably expose both here. See the method `miner_nominal_power_meets_consensus_minimum` in state.rs. We can return`(claimed-power, meets-consensus-minimum?)`. Then we should have network_raw_power equal to the sum of miner power where the bool is true.

Reply:
Yeah, I think I'm happy to go with returning both. I am a little unsure how useful it is to share `meets-consensus-minimum`, but it's worth doing.